vendor/views/vendor_admin.py

```diff
@@ -119,30 +119,38 @@ def get_context_data(self, **kwargs):
 
         context['customers_who_own'] = customers_who_own
         context['customers_who_dont_own'] = customers_who_dont_own
+        
         context['formset'] = PriceFormSet(instance=self.object)
-        return context
 
+        return context
 
     def form_valid(self, form):
         price_formset = PriceFormSet(self.request.POST, self.request.FILES, instance=Offer.objects.get(uuid=self.kwargs['uuid']))
 
-        if not (price_formset.is_valid() or form.is_valid()):
+        if not form.is_valid():
             return render(self.request, self.template_name, {'form': form, 'formsert': price_formset})
```

Comment:
may have misspelled the formset ket. you have `formsert` 👎

Reply:
Lines removed as they were not required to double validate form.is_valid() given that we are on form_valid call jajajaj